app/views/about-your-organisation/placement-policy.html

```diff
@@ -18,29 +18,29 @@
         items: [{
           value: "hosted",
           text: "Candidates choose a lead or host school",
-          checked: checked((code + "-placement-policy"), "hosted"),
+          checked: checked(("placement-policy"), "hosted"),
           hint: {
             text: "Placement schools are shown on Find with their vacancy status. Candidates will be asked to select a school when they apply."
           }
         }, {
           value: "placed",
           text: "Placements are arranged with candidates before their training starts",
-          checked: checked((code + "-placement-policy"), "placed"),
+          checked: checked(("placement-policy"), "placed"),
           hint: {
             text: "Candidates apply directly to the course when they apply."
           }
         }, {
           value: "other",
           text: "Other",
-          checked: checked((code + "-placement-policy"), "other")
+          checked: checked(("placement-policy"), "other")
         }]
       }) }}
 
       {% if change %}
         {{ govukButton({
           text: "Save and publish changes"
         }) }}
-        <p class="govuk-body"><a href="{{ paths.back }}">Cancel changes</a></p>
+        <p class="govuk-body"><a href="{{ backLink }}">Cancel changes</a></p>
```

Comment:
✂️  this? Having a cancel link so close to a save button feels dangerous - and we have a back link.

Reply:
This is the same as what we have on the other pages in ‘About your organisation’, so will leave as is. We can (and should) address these other usability issues separately, though.